source/MaterialXTest/ShaderGen.cpp

```diff
@@ -19,6 +19,7 @@
 #include <fstream>
 #include <iostream>
 #include <vector>
+#include <set>
```

Comment:
Missing include made build fail on mac and linux. Also changed from a list to a set to simplify the logic below.

Reply:
Thanks for this fix.